core/services/bulletprooftxmanager/eth_broadcaster.go

```diff
@@ -162,6 +163,14 @@ func (eb *ethBroadcaster) ProcessUnstartedEthTxs(key models.Key) error {
 // First handle any in_progress transactions left over from last time.
 // Then keep looking up unstarted transactions and processing them until there are none remaining.
 func (eb *ethBroadcaster) processUnstartedEthTxs(fromAddress gethCommon.Address) error {
+	var n uint = 0
+	mark := time.Now()
+	defer func() {
+		if n > 0 {
+			logger.Debugw("EthBroadcaster: finished processUnstartedEthTxs", "address", fromAddress, "time", time.Since(mark), "n", n, "id", "eth_broadcaster")
```

Comment:
I think this `n` variable could use a better name! It will also help when reading the log line.
Also, re. the `"time"` tag, I suggest you rename it to `duration` (or `time_spent`) and have the value converted to a more human readable form - something like https://pkg.go.dev/time?tab=doc#Duration.String. You can use this trick in other places in this PR!

Reply:
`time.Since` returns a duration which is automatically formatted into a string as you have described.